etcd/install.sls

```diff
@@ -9,17 +9,10 @@ include:
 
   {%- if etcd.manage_users %}
 etcd-user-group-home:
-  group.present:
-    - name: {{ etcd.group or 'etcd' }}
-    - system: True
-    - require_in:
-      - user: etcd-user-group-home
```

Comment:
Why is this section being removed?  I don't use this formula so I can't be sure if this should be done or not.  If it should be, then it would be worth mentioning the rationale in the PR's main comment, so that future developers can understand what the thought process was.

Reply:
Group is referenced here: https://github.com/saltstack-formulas/etcd-formula/blob/d4c849a76f1bfa621545c82b28b8c45b2ee6d070/etcd/install.sls#L40